src/reducers/index.ts

```diff
@@ -0,0 +1,27 @@
+import { combineReducers } from 'redux';
+
+import customLocation from './custom-location';
+import dataProgress from './data-progress';
+import download from './download';
+import favourites from './favourites';
+import jobs from './jobs';
+import settings from './settings';
+import status from './status';
+import toasts from './toasts';
+import ui from './ui';
+
+// TODO: If nothing breaks by InterPro 102.0 Delete the inport and the utility class
```

Comment:
yeah, I check around and is not used anywhere else, so unless Im missing something it can be deleted, but postponing it for at least one release in case I oversaw something.